### PR TITLE
WIP: Revert the use of _ to separate the path element in keys for the Ax sweeper plugin.

### PR DESCRIPTION
**Pull request by shagunsodhani**
## Motivation

Revert the use of _ to separate the path element in keys for the Ax sweeper plugin.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes

## Test Plan

CI

## Related Issues and PRs

#913 
(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

**Comment by shagunsodhani** [comment]
This is a WIP PR, to check all tests pass on the CI. Once https://github.com/facebookresearch/hydra/pull/912 lands, I will rebase and request review.

**Comment by omry** [comment]
Please give a better title.

File: news/913.plugin [hunk 1]
```diff
@@ -0,0 +1 @@
+Revert the use of "_" to separate the path element in keys for the Ax sweeper plugin.
```
Comment (omry) [inline news/913.plugin:1]: This is a breaking change. use api_change file extension (see pyproject.toml for available news types).

File: news/870.plugin [hunk 2]
```diff
@@ -0,0 +1 @@
+Switch Ax parser to use new Hydra command line syntax
```
Comment (omry) [inline news/870.plugin:1]: Is this diff doing both? why?
if not, what is this news fragment doing here?

Comment (shagunsodhani) [reply news/870.plugin:1]: Hey :) This is a WIP PR, to check all tests pass on the CI. Once #912 lands, I will rebase and request review.

**Comment by omry** [comment]
can you convert it to draft?

**Comment by shagunsodhani** [comment]
Windows test failed with "ssh: connect to host github.com port 22: Connection timed out". Will rerun the test after rebasing.

**Comment by shagunsodhani** [comment]
Closing in favor of https://github.com/facebookresearch/hydra/pull/1014

**Pull request closed without merging**